### PR TITLE
Creating base structure

### PR DESCRIPTION
**Pull request by ajakubo1**
Creating a webpack configuration for our react application + adding a possibility of writing tests.

All of the work based on survivejs webpack tutorial:
https://survivejs.com/webpack/introduction/

I didn't configure fully the react side of things, I've stopped here: http://survivejs.com/webpack/advanced-techniques/configuring-react/#setting-up-babel-preset-react-hmre-

I'm not sure we would need other features for now :).

I used mocha and nyc for tests/coverage.

I think it would be nice if we also added eslint + config for it, but it's not badly required as well ;).

A heads up: I didn't deal with real application code or any geolocation api for now, so there's not much to browse on the webpage :D. But I think it's a good place to start with development.

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+.node_repl_history
+
+# Output of 'npm pack'
+*.tgz
+
+# Yarn Integrity file
+.yarn-integrity
+
+#some leftover files after running mocha
+.tmp
```
Comment (lechup) [inline .gitignore]: New lines new lines... ;)

**Review by lechup — APPROVED** [review]
Generally looking good (only new lines ;)) I'll prepare docker setup for development based on that branch .

I think this PR is just as @ajakubo1 said just base. So for sure it will evolve.

PS: Do we have travis setup or anything similar for CI?  Just added #6...

File: .gitignore (hunk 1 above)
Comment (ajakubo1) [reply .gitignore]: Oh yeah - maybe eslint is more needed then I thought :D.

**Comment by ajakubo1** [comment]
OK, I've added the newlines. I'll merge the branch if it looks OK to you @lechup ;).

**Pull request merged**